test/e2e/data/infrastructure-docker/v1beta1/cluster-template-upgrades-runtimesdk/cluster-runtimesdk.yaml

```diff
@@ -3,6 +3,27 @@ kind: Cluster
 metadata:
   name: '${CLUSTER_NAME}'
   namespace: default
+  labels:
+    cni: "${CLUSTER_NAME}-crs-0"
 spec:
+  clusterNetwork:
+    services:
+      cidrBlocks: ['${DOCKER_SERVICE_CIDRS}']
+    pods:
+      cidrBlocks: ['${DOCKER_POD_CIDRS}']
+    serviceDomain: '${DOCKER_SERVICE_DOMAIN}'
   topology:
```

Comment:
Should we add nodeDeletionTimeouts here as well? Just to keep the intentional diff to cluster-with-topology.yaml as small as possible?